### PR TITLE
Germline

### PR DESCRIPTION
**Pull request by MaxUlysse**
- removed unused or useless channel operations

## PR checklist
 - [X] PR is made against `dev` branch
 - [X] This comment contains a description of changes (with reason)
 - [X] Ensure the test suite passes (`./scripts/test.sh -p docker -t ALL`).
 - [X] `CHANGELOG.md` is updated

### CODE REVIEW

File: germlineVC.nf [hunk 1]
```diff
@@ -91,52 +91,26 @@ if (params.verbose) bamFiles = bamFiles.view {
...
-  .choice(bamsTumor, bamsNormal) {it[1] == 0 ? 1 : 0}
+bamsToKeep = Channel.create()
+bamsOthers = Channel.create()
 
-// Ascat, Strelka Germline & Manta Germline SV
-bamsForAscat = Channel.create()
-bamsForSingleManta = Channel.create()
-bamsForSingleStrelka = Channel.create()
+bamFiles
+  .choice(bamsOthers, bamsToKeep) {it[1] == 0 ? 1 : 0}
```
Comment (alneberg) [inline germlineVC.nf]: How does this work? I Have no idea what's going on here...

Comment (MaxUlysse) [reply germlineVC.nf]: If the status of a set is 0 -> bamsToKeep, else -> bamsOthers
I know that this choice operator can be confusing, especially with no comments.
I'll had some ;-)

Comment (alneberg) [reply germlineVC.nf]: Ah, I see. You're checking the status field in the tsv. That's the part I was missing.

Comment (MaxUlysse) [reply germlineVC.nf]: Just added comments, I hope it'll be easier to understand

**Comment by MaxUlysse** [comment]
I removed HaplotypeCaller and Strelka for tumor sample, I don't think it made much sense to have that.
What do you think?
Shoul I ask arroung if it was interesting or not?

File: germlineVC.nf (hunk 1 above)
Comment (szilvajuhos) [reply germlineVC.nf]: thank you indeed :)

**Comment by szilvajuhos** [comment]
GVCFs are certainly not needed, we planned to use the tumor germlines for
QC - i.e. checking whether samples are mixed. Looks Markus is using HC
tumor VCF in pathfindr.
Szilva

**Comment by MaxUlysse** [comment]
OK, I'll keep Strelka and HC then just in case

File: germlineVC.nf (hunk 1 above)
Comment (MaxUlysse) [reply germlineVC.nf]: Not doing that anymore...
it's now more efficient, and easier to understand I think

**Pull request merged**